zgrab2/multiple.go

```diff
@@ -0,0 +1,31 @@
+package zgrab2
+
+import (
+	"errors"
+	"os"
+
+	"github.com/ajholland/zflags"
+)
+
+type MultipleConfig struct {
+	ConfigFileName  string `short:"c" long:"config-file" default:"-" description:"Config filename, use - for stdin"`
+	ContinueOnError bool   `long:"continue-on-error" description:"If proceeding protocols error, do not run following protocols (default: true)"`
+}
+
+// Validates the options sent to MultipleConfig, and parses the configFile
+func (x *MultipleConfig) Validate(args []string) error {
```

Comment:
This config stuff is getting really hard to keep track of. What's the rationale against having a single config struct, a single `ConfigFileName` string, and then just reading the file at the start if passed?